mastering-plone/dexterity_2.rst

```diff
@@ -436,7 +436,7 @@ Query for custom indexes
 
 The new indexes behave like the ones that Plone has already built in:
 
-.. code-block:: pycon
+.. code-block:: python
```

Comment:
pycon is python console, and is correct. Please revert. See http://pygments.org/docs/lexers/#pygments.lexers.python.PythonConsoleLexer